### ISSUE TITLE
[A11y]Screenreader does not read required keyword for the control “Package Owner".

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8494]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1296227

​## Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)
Screen reader version: JAWS 2020,  NVDA 2020.2 .

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Select “API Keys” from the Profile Dropdown and hit enter.
"API Keys" screen gets opened.
Start screenreader and navigate to package owner filed and observe screenreader announce the required keyword or not.
## Actual:​
Screenreader does not read required keyword for the control “Package Owner".

This issue is observed with both JAWS and NVDA.
## Expected:​
Screenreader should announce the required keyword for the control “Package Owner".
​## User Impact:​
Screenreader user will not get proper information regarding cell values belongs to which column header.

### Notes
This should (hopefully) be as simple as ensuring that whatever text is providing the information here includes the word "Required"
